bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/thing/channelTypes.xml

```diff
@@ -184,6 +204,9 @@
 		<label>@text/channel-type.mielecloud.finish_state.label</label>
 		<description>@text/channel-type.mielecloud.finish_state.description</description>
 		<category>Alarm</category>
+		<tags>
+			<tag>Alarm</tag>
```

Comment:
I guess the `category` is only set to reference a specific icon (not sure). This is not actually an alarm, just an indication that the program is finished ("Indicates whether the most recent program finished.")

Reply:
So `Status:Info` ??